src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -709,13 +709,17 @@ public StudentParticipation findOneByExerciseIdAndStudentLogin(Long exerciseId,
     }
 
     /**
-     * Get all participations belonging to course with relevant results.
+     * Get all participations of submissions that are submitted and do not already have any result.
```

Comment:
Is this a merge conflict? Please restore the  JavaDoc comment for the `findByCourseIdWithRelevantResults` method.

Reply:
Done